scripts/data/get_blocks.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+set -e;
+set -o pipefail;
+
+get_single_block() {
+    local block_hash=$1
+    ./scripts/data/get_block.sh "$block_hash"
+}
+
+main() {
+    local block_hashes=(
+        "000000000019d6689c085ae165831e934ff763ae46a2a6c172b3f1b60a8ce26f"  # Genesis block
+        "00000000d1145790a8694403d4063f323d499e655c83426834d4ce2f8dd4a2ee"  # Block 170
```

Comment:
Are you going to add more blocks?

Reply:
i was thinking of opening a dedicated PR for this

Reply:
as your prefer!

Reply:
I prefer another pr.